crossdock/main.go

```diff
@@ -105,7 +105,7 @@ func (h *clientHandler) isInitialized() bool {
 func httpHealthCheck(logger *zap.Logger, service, healthURL string) {
 	for i := 0; i < 240; i++ {
 		res, err := http.Get(healthURL)
-		if err == nil && res.StatusCode == 204 {
+		if err == nil && res.StatusCode == 200 {
```

Comment:
I'd say that the most appropriate is to check if the response is a 2xx class. That's what monitoring solutions do.